DonutsGo/DonutsGo.API/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using DonutsGo.Application.Exceptions;
+using DonutsGo.Application.Models.Products;
+using DonutsGo.Application.Services;
+using DonutsGo.DataAccess;
+using DonutsGo.DataAccess.Entities;
+using Microsoft.AspNetCore.Mvc;
+
+
+
+
+namespace DonutsGo.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ProductsController : ControllerBase
+    {
+        private readonly IProductService productService;
+
+        public ProductsController(IProductService productService)
+        { 
+            this.productService = productService;
+        }
+
+        [HttpGet]
+        public IActionResult GetProducts()
+        {
+          
```

Comment:
Remove empty line, please. It is useless and makes code reading more complicated.